techeria_controller/techeria_app/models.py

```diff
@@ -4,13 +4,13 @@
 # Create your models here.
 
 class Products(models.Model):
+    image = models.ImageField(null=False, blank=False, upload_to='images/')
     name = models.CharField(max_length=255)
     price = models.FloatField()
-    description = models.CharField(max_length=255)
```

Comment:
Can we put textfield [description = models.TextField()] we can specify text length too for simplicity